messages.rb

```diff
@@ -276,6 +289,92 @@ def send_list(event)
   event.attach_file(File::open(tmpfile))
 end
 
+def send_community(event)
+  msg = event.content
+  tabs = parse_tabs(msg)
+  tabs.empty? ? ftabs = "" : ftabs = format_tabs(tabs)
+  tabs = (tabs.empty? ? [:SI, :S, :SL, :SU, :SS, :SS2] : tabs)
+
+  levels = Score.where(rank: 0, highscoreable_type: "Level").includes(:level).where(levels: {tab: tabs})
+  episodes = Score.where(rank: 0, highscoreable_type: "Episode").includes(:episode).where(episodes: {tab: tabs})
+  lvls = levels.count
+  eps = episodes.count
+  tls = levels.map{|s| s.score}.sum
```

Comment:
1. I prefer spaces around blocks like this as well, `map { |s|` instead of `map{|s|`.
2. `tls` and `tes` is a bit challenging to understand - total level/episode score, presumably? Maybe just `level_total`/`episode_total` instead?

Reply:
Gotcha, I'll keep that spacing in mind ;)
Also updated all those weird variable names.